.travis.yml

```diff
@@ -12,6 +12,7 @@ addons:
       - re2c
       - ccache
       - mysql-server
+      - libavif-dev
```

Comment:
It looks like this package is only available in Ubuntu 21.04 :/

Reply:
Good point, @nikic . I wasn't sure what to do here. Because, indeed, when we made [the libgd PR](https://github.com/libgd/libgd/pull/671), we ended up [requiring libavif >= v.0.8.2](https://github.com/libgd/libgd/blob/82d260950589563a1af9c56f4ce5fde843a695ae/CMakeLists.txt#L129).

For Ubuntu, as you say, that's available [in hirsute and up](https://packages.ubuntu.com/search?suite=hirsute&keywords=libavif).

Unfortunately, also, the most recent version of Ubuntu [travis supports](https://docs.travis-ci.com/user/reference/linux/) is focal (20.04). In libgd, we ended up [just skipping AVIF support](https://github.com/libgd/libgd/pull/671#issuecomment-785379005) in test coverage until travis catches up.

To use bionic, we'd need to rewrite this code to use libavif v0.7 instead.

This is my first PHP PR, so I can only appeal to your superior experience to decide what to do here :)

Reply:
I think you can drop avif testing from travis and amd64/i386 azure, and maybe instead add it to the recently added `.cirrus.yml`. That one uses FreeBSD 13, and it looks like that it should provide libavif 0.9.

Possibly azure macos would also work -- depends on whether homebrew provides it.

Reply:
Thanks. I'll rebase on the latest `master` to pick that up, and I'll give it a try.